develop/tutorials/articles/100-tooling/04-liferay-workspace/09-updating-default-plugins-provided-by-workspace.markdown

```diff
@@ -0,0 +1,49 @@
+# Updating Default Plugins Provided by Liferay Workspace
+
+Liferay Workspace is bundled with a slew of plugins that provide many features.
+For example,
+
+- [CSS Builder](https://github.com/liferay/liferay-portal/tree/master/modules/util/css-builder)
+- [Javadoc Formatter](https://github.com/liferay/liferay-portal/tree/master/modules/util/javadoc-formatter)
+- [Lang Builder](https://github.com/liferay/liferay-portal/tree/master/modules/util/lang-builder)
+- [Service Builder](https://github.com/liferay/liferay-portal/tree/master/modules/util/portal-tools-service-builder)
+- [Source Formatter](https://github.com/liferay/liferay-portal/tree/master/modules/util/source-formatter)
+- [Theme Builder](https://github.com/liferay/liferay-portal/tree/master/modules/util/portal-tools-theme-builder)
+- etc.
+
+The bundled plugins are updated with each release of workspace, but what if you
+want workspace to use a different version of a bundled plugin? For example,
+suppose a new feature is developed for the
+[Source Formatter plugin](https://github.com/liferay/liferay-portal/tree/master/modules/util/source-formatter),
+but the latest workspace version has not yet been updated to include it.
+
+To upgrade one of workspace's bundled plugins, follow these steps:
+
+1.  Find the bundle symbolic name (BSN) for the bundled plugin you want to
```

Comment:
Instead of this, we should instruct them to look into this file

https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins/src/main/resources/com/liferay/gradle/plugins/dependencies/portal-tools.properties

This shows the possible versions they can get in their build.gradle.